cardigan/stories/components/Cards/Cards.stories.tsx

```diff
@@ -106,6 +111,22 @@ exhibitionPromo.parameters = {
 };
 exhibitionPromo.storyName = 'ExhibitionPromo';
 
+const GuideStopCardTemplate = args => <GuideStopCard {...args} />;
+export const guideStopCard = GuideStopCardTemplate.bind({});
+guideStopCard.args = {
+  link: '/stop-1',
+  totalStops: 3,
+  duration: 3.5,
+  number: 1,
+  title: 'Exhibition guide stop',
+  type: 'audio',
+  image: imageWithCrops,
+};
+guideStopCard.parameters = {
+  ...sharedParameters,
+};
+guideStopCard.storyName = 'GuideStopCard';
+
```

Comment:
optional: 

You could make this story more interactive if you added 
```
guideStopCard.argTypes = {
  type: {
    options: ['audio', 'video'],
    control: { type: 'radio' },
  }
};
```

To allow anyone (like designers) to see what it changes when the other type is selected. Could also be a dropdown (Wanted to link to storybook docs but they seem partly down at the mo? We have examples in other stories).

To make it even nicer, I'd do
```
guideStopCard.argTypes = {
  type: {
    options: ['audio', 'video'],
    control: { type: 'radio' },
  },
  link: {
    table: {
      disable: true,
    },
  },
  image: {
    table: {
      disable: true,
    },
  },
};
```

Which will remove the props we don't care about seeing/changing. This is a basic/random selection of them, but that's how to do it. Looks like:

<img width="572" alt="Screenshot 2024-08-07 at 16 53 52" src="https://github.com/user-attachments/assets/329e728f-369d-493d-9134-ec270f591cc5">

Reply:
good shout – I'll do this